### PR TITLE
(1b) Dive summary, requested changes 

### PR DESCRIPTION
**Pull request by janiversen**
### Describe the pull request:

- [ x] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Add icon to be used in global drawer.
Correct SAC calculation.

### CODE REVIEW

File: mobile-widgets/qml/icons/sigma.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN"
+"http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<!-- Created with Sodipodi ("http://www.sodipodi.com/") -->
```
Comment (dirkhh) [inline mobile-widgets/qml/icons/sigma.svg:4]: so we borrowed a lot of our icons from Breeze and from the Google Material Design icon set.
Did you go straight for your own design or did you look if one of those two sources have something reasonable to use?
I'm fine with a custom icon (haven't looked at this one, I'm on a plane), but I wonder if we should try to find something by the same design team, first, before doing our own.
Again, apologies for not looking at it before commenting - maybe this is perfect :-)

**Review by dirkhh — APPROVED** [review]
See my small comment about the icon.
Also, this doesn't appear to depend on (3), so once I had a look at the icon and assuming I love it, I should be able to merge this before we merge (3), right?

**Comment by dirkhh** [comment]
The icon is ok, not great. We can always replace it later.
My plane is about to land, so I'll be offline for a while, but I might merge this locally while sitting in the airport waiting for my transatlantic flight... :-)

File: mobile-widgets/qml/icons/sigma.svg (hunk 1 above)
Comment (janiversen) [reply mobile-widgets/qml/icons/sigma.svg:4]: I did not find the sigma symbol, but maybe they named it differently.

and no it does not depend on (3) solely (1) and (2).

**Comment by janiversen** [comment]
Going to Europe ? or the soon to be non-Europe (UK).

I will not touch this, I assume your transatlantic flight will start before I wake up again.

**Comment by dirkhh** [comment]
on my way to Bulgaria and then Germany. I merged this and pushed it. I'll try to find time to look for a different icon, but as I said, this is fine in the meantime.

We got a lot of stuff merged in the last few days, Jan...

**Comment by janiversen** [comment]
"We got a lot of stuff merged in the last few days, Jan..."

We sure did, and not just changes, but a lot of changed user experience as well, which remind me, at some point the manual needs an update.

Just maybe, this contributor does some serious work, independent of the very low image in this project (haven't had my morning coffee yet, should I need an excuse for this sentence).

**Pull request closed without merging**